common/src/main/java/com/zegelin/prometheus/netty/Server.java

```diff
@@ -62,27 +70,44 @@ public static void start(final List<InetSocketAddress> listenAddresses, final Ha
                 .channel(NioServerSocketChannel.class)
                 .childHandler(new ChildInitializer(harvester, helpExposition));
 
-        final List<Channel> channels;
         {
             final ImmutableList.Builder<Channel> builder = ImmutableList.builder();
 
             for (final InetSocketAddress listenAddress : listenAddresses) {
-                final Channel channel = bootstrap.bind(listenAddress).sync().channel();
-
-                builder.add(channel);
+                builder.add(bootstrap.bind(listenAddress).sync().channel());
             }
 
-            channels = builder.build();
+            Server.channels = builder.build();
         }
 
         if (logger.isInfoEnabled()) {
-            logger.info("cassandra-exporter started. Listening on {}", Joiner.on(", ").join(
+            logger.info("Cassandra Exporter Agent Server has started. Listening on {}", Joiner.on(", ").join(
```

Comment:
This is shared between both the agent and standalone versions -- probably best to just leave it as the original message.

Reply:
Or "... HTTP server has..."?